cluster/charts/crossplane/values.yaml

```diff
@@ -1,213 +1,218 @@
+# -- Add custom `labels` to the Crossplane pod deployment.
 customLabels: {}
 
-# -- Custom annotations to add to the Crossplane deployment and pod.
+# -- Add custom `annotations` to the Crossplane pod deployment.
 customAnnotations: {}
 
 serviceAccount:
-  # -- Custom annotations to add to the serviceaccount of Crossplane.
+  # -- Add custom `annotations` to the Crossplane ServiceAccount.
   customAnnotations: {}
 
-# -- Enable leader election for Crossplane Managers pod.
+# -- Enable leader election for the Crossplane pod.
 leaderElection: true
-# -- A list of additional args to be passed to Crossplane's container.
+# -- Add custom arguments to the Crossplane pod.
 args: []
 
 provider:
-  # -- The list of Provider packages to install together with Crossplane.
+  # -- A list of Provider packages to install.
   packages: []
 
 configuration:
-  # -- The list of Configuration packages to install together with Crossplane.
+  # -- A list of Configuration packages to install.
   packages: []
 
-# -- Names of image pull secrets to use.
+# -- The imagePullSecret names to add to the Crossplane ServiceAccount.
 imagePullSecrets: {}
 
 registryCaBundleConfig:
-  # -- Name of ConfigMap containing additional CA bundle for fetching from package registries.
-  name: {}
-  # -- Key to use from ConfigMap containing additional CA bundle for fetching from package registries.
-  key: {}
+  # -- The ConfigMap name containing a custom CA bundle to enable fetching packages from registries with unknown or untrusted certificates.
+  name: ""
+  # -- The ConfigMap key containing a custom CA bundle to enable fetching packages from registries with unknown or untrusted certificates.
+  key: ""
 
 webhooks:
-  # -- Enable webhook functionality for Crossplane as well as packages installed by Crossplane.
+  # -- Enable webhooks for Crossplane and installed Provider packages.
   enabled: true
 
 rbacManager:
-  # -- Deploy RBAC Manager and its required roles.
+  # -- Deploy the RBAC Manager pod and its required roles.
   deploy: true
-  # -- Opt out of deploying aggregated ClusterRoles.
+  # -- Don't install aggregated Crossplane ClusterRoles.
   skipAggregatedClusterRoles: false
-  # -- The number of replicas to run for the RBAC Manager pods.
+  # -- The number of RBAC Manager pod `replicas` to deploy.
   replicas: 1
-  # -- The extent to which the RBAC manager will manage permissions:.
-  # - `All` indicates to manage all Crossplane controller and user roles.
-  # - `Basic` indicates to only manage Crossplane controller roles and the `crossplane-admin`, `crossplane-edit`, and `crossplane-view` user roles.
+  # -- Defines the Roles and ClusterRoles the RBAC Manager creates and manages.
+  # - A policy of `Basic` creates and binds Roles only for the Crossplane ServiceAccount, Provider ServiceAccounts and creates Crossplane ClusterRoles.
+  # - A policy of `All` includes all the `Basic` settings and also creates Crossplane Roles in all namespaces.
```

Comment:
My only doubt is whether we should explicitly mention the roles names as we did before here.

Reply:
To make it clearer, I meant mentioning explicitly`crossplane-admin`, `crossplane-edit`, and `crossplane-view` as we were doing before.